File: src/Web/ClientApp/src/app/core/api-routes.ts
/** Must match {@link Ntk.Note.IP.Web.Infrastructure.ApiRoutes.VersionPrefix}. */
export const API_V1_PREFIX = '/api/v1';

/** Short alias for GetMyIpPlain — must match {@link Program} MapGet("/myIp"). */
export const MY_IP_SHORT_PATH = '/myip';

export function apiV1Group(groupName: string): string {
  return `${API_V1_PREFIX}/${groupName}`;
}

File: src/Mobile/ntk_note_ip_app/lib/core/api/api_routes.dart
abstract final class ApiRoutes {
  /// Short alias for GetMyIpPlain — must match Program.cs MapGet("/myIp").
  static const myIpShortPath = '/myip';

  static String myIpPlainUrl(String apiBaseUrl) {
    final base = apiBaseUrl.endsWith('/')
        ? apiBaseUrl.substring(0, apiBaseUrl.length - 1)
        : apiBaseUrl;
    return '$base$myIpShortPath';
  }
}
